packages/dd-trace/test/appsec/index.express.plugin.spec.js

```diff
@@ -96,20 +107,65 @@ withVersions('express', 'express', version => {
         appsec.disable()
       })
 
-      it('should get the schema', async () => {
-        appsec.enable(config)
+      describe('with requestSampling 1.0', () => {
+        beforeEach(() => {
+          config.appsec.apiSecurity.requestSampling = 1.0
+          appsec.enable(config)
+        })
 
-        const expectedSchema = zlib.gzipSync(JSON.stringify([{ 'key': [8] }])).toString('base64')
-        const res = await axios.post(`http://localhost:${port}/`, { key: 'value' })
+        it('should get the schema', async () => {
+          const expectedRequestBodySchema = zlib.gzipSync(JSON.stringify([{ 'key': [8] }])).toString('base64')
```

Comment:
```javascript
function formatSchema (obj) {
  return zlib.gzipSync(JSON.stringify(obj)).toString('base64')
}
```

Reply:
You're right, I don't have the habit to reuse code in tests... and I should